lib/bootstrap.js

```diff
@@ -14,22 +14,20 @@ const isCommandUsed = commands =>
 
 async function runCLI(cli, commandIsUsed) {
     let args;
-    const helpFlagExists = isFlagPresent(process.argv, 'help');
```

Comment:
I would revert this change. As it is at the moment it's better because it's possible to test the function and it's a better functional approach where the function stays pure.